fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/codec/LuceneOptimizedStoredFieldsFormatTest.java

```diff
@@ -73,17 +70,16 @@ private static boolean isUsingFDBDirectory() {
     public void setUp() throws Exception {
         super.setUp();
         TestingCodec.reset();
-        // We have to manually call the extension call backs because this is a JUnit4 class
-        dbExtension.beforeEach(null);
         TestFDBDirectory.reset();
+        BaseIndexFileFormatTestCaseUtils.resetStaticConfigs();
```

Comment:
Calling `resetStaticConfigs` means that this doesn't need to explicitly call `TestingCodec.reset()` or `TestFDBDirectory.reset()`. Same goes for the `tearDown()`